config/bidderinfo.go

```diff
@@ -403,36 +405,32 @@ func validateMaintainer(info *MaintainerInfo, bidderName string) error {
 	return nil
 }
 
-func validateAliasCapabilities(bidder BidderInfo, infos BidderInfos, bidderName string) error {
-	if len(bidder.AliasOf) == 0 {
-		return nil
-	}
-
-	parentBidder, parentFound := infos[bidder.AliasOf]
+func validateAliasCapabilities(aliasBidderInfo BidderInfo, infos BidderInfos, bidderName string) error {
+	parentBidder, parentFound := infos[aliasBidderInfo.AliasOf]
 	if !parentFound {
-		return fmt.Errorf("bidder: %s not found for an alias: %s", bidder.AliasOf, bidderName)
+		return fmt.Errorf("bidder: %s not found for an alias: %s", aliasBidderInfo.AliasOf, bidderName)
```

Comment:
Can you add a test to cover this error case?

Reply:
Added in [0db8d10](https://github.com/prebid/prebid-server/pull/2996/commits/0db8d100c06228c2360c7b19b4500a3dbb8a4c54)